pkg/producers/opslog/prometheus_sli_test.go

```
@@ -0,0 +1,101 @@
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package opslog
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	dto "github.com/prometheus/client_model/go"
+)
+
+// collectToDTO is a test helper that collects all metrics and returns them as dto.Metric slices.
+func (c *sliCollector) collectToDTO() []*dto.Metric {
```

golangci-lint (2.12.2)

[error] func (*sliCollector).collectToDTO is unused

(unused)